core/src/main/java/org/mule/api/config/MuleConfiguration.java

```diff
@@ -52,6 +52,16 @@
     int getShutdownTimeout();
 
     /**
+     * The approximated maximum space in megabytes used by the transaction log files for transactional persistent queues.
+     *
+     * Take into account that this number applies both to the set of transaction log files for XA and for local transactions.
+     * If both type of transactions are used then the approximated maximum space used will be twice the configured value.
+     *
+     * @return the approximated maximum space in disk that the transactions logs can use in megabytes.
+     */
+    Integer getQueueTransactionFilesSizeInMegabytes();
```

Comment:
Is it possible to pull the current default up to DefaultMuleConfiguration and therefore return int here instead of Integer?